src/features/author.ts

```diff
@@ -0,0 +1,13 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+
+const authorSlice = createSlice({
+  name: 'author',
+  initialState: null as User | null,
+  reducers: {
+    setUser: (user, { payload }: PayloadAction<User | null>) => payload,
```

Comment:
Consider adding types to your function parameters. This will improve type safety and make the code easier to understand. For example, you can type `user` as `User | null`.